website/content/docs/job-specification/service.mdx

```diff
@@ -15,10 +15,10 @@ description: |-
   ]}
 />
 
-The `service` stanza instructs Nomad to register a service with Consul. This
-section of the documentation will discuss the configuration, but please also
-read the [Nomad service discovery documentation][service-discovery] for more
-detailed information about the integration.
+The `service` stanza instructs Nomad to register a service with the specified
+provider. This section of the documentation will discuss the configuration, but
```

Comment:
maybe put "(Nomad or Consul)" after "the specified provider" since people wont know what that means?